apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -307,6 +307,10 @@ export default function SectionsSetUpContainer({
   };
 
   const renderCoteacherSection = () => {
+    const isCoTeacherManagementDisabled =
```

Comment:
Nice, this also helps contain the camelized `ltiV1` string to one spot.

Reply:
I was just about to comment on trying to at least move the `ltiV1` string to a variable, but this fixes it! Nice 👍